app/views/admin/worldwide_organisations/index.html.erb

```diff
@@ -7,6 +7,12 @@
     text: "Create worldwide organisation",
     href: new_admin_worldwide_organisation_path,
     margin_bottom: 6,
+    data_attributes: {
+      module: "gem-track-click",
+      "track-category": "form-button",
+      "track-action": "world-wide-organisation-create-button",
+      "track-label": "Create worldwide organisation"
+    }
```

Comment:
I'm not familiar with tracking categories so this may be fine but a couple of thoughts: 
- Are we even tracking the "Create" action? I don't think we have done elsewhere. 
- It isn't strictly speaking a form button
- The action should probably be "world-wide-organisation-button"

Reply:
i think it would be  `worldwide-organisation-button` as the model is called WorldwideOrganisation